akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/javadsl/TestKitJunitResource.scala

```diff
@@ -143,4 +143,9 @@ final class TestKitJunitResource(_kit: ActorTestKit) extends ExternalResource {
     testKit.shutdownTestKit()
   }
 
+  /**
+   * See corresponding method on [[ActorTestKit]]
+   */
+  def verifySerialization[M](obj: M, assertEquality: Boolean): M = testKit.verifySerialization(obj, assertEquality)
```

Comment:
Just a style thing, line 211 `verifySerialization` above is split into two lines but could be one, and this is one. I'd pick one for a commit.

Reply:
this has been fixed, moved to separate SerializationTestKit